### PR TITLE
Fix existing: Extrude from command bar selects extrude line after

### LINKED ISSUE #3545: Fix existing: Extrude from command bar selects extrude line after

### LINKED ISSUE #3546: Fix existing: Fillet from command bar

### PR DESCRIPTION
**Pull request by Irev-Dev**
Resolves  #3545

Oh, it resolves #3546 too.

### CODE REVIEW

File: src/components/CommandBar/CommandBarKclInput.tsx [hunk 1]
```diff
@@ -84,17 +83,13 @@ function CommandBarKclInput({
...
       }),
       varMentions(varMentionData),
-      EditorState.transactionFilter.of((tr) => {
-        if (tr.newDoc.lines > 1) {
-          handleSubmit()
-          return []
-        }
-        return tr
-      }),
       EditorView.updateListener.of((vu: ViewUpdate) => {
```
Comment (Irev-Dev) [inline src/components/CommandBar/CommandBarKclInput.tsx:93]: I might get a review from you @franknoirot, I don't know why this transaction stopped working in the migration, but I figured listening for keydown was okay.

**Comment by lf94** [comment]
I'll fix the other tests tomorrow. @franknoirot caught it's probably `useAbsoluteFilePath` changed causing the others to break. I think I might have a workaround but we'll see tomorrow.

File: src/components/CommandBar/CommandBarKclInput.tsx [hunk 2]
```diff
@@ -107,7 +102,7 @@ function CommandBarKclInput({
     if (editorRef.current) {
       setContainer(editorRef.current)
     }
-  }, [arg, editorRef])
```
[the remark this answers is not in the document]
Comment (Irev-Dev) [reply src/components/CommandBar/CommandBarKclInput.tsx]: 473 to go.

File: src/components/CommandBar/CommandBarKclInput.tsx (hunk 1 above)
Comment (franknoirot) [reply src/components/CommandBar/CommandBarKclInput.tsx:93]: Yeah I'm fairly confident that this is okay to do. This plus some issues that @iterion has been seeing with his Windows fix PR wrt the CodeMirror instance make me nervous that something is up with CodeMirror right now, so I just want to flag that.

This change looks G2G though.

**Review by franknoirot — APPROVED** [review]

**Pull request merged**